src/transforms/groupby.js

```diff
@@ -137,8 +137,22 @@ function pasteArray(newTrace, trace, j, a) {
     );
 }
 
+// In order for groups to apply correctly to other transform data (e.g.
+// a filter transform), we have to break the connection and clone the
+// transforms so that each group writes grouped values into a different
+// destination. This function does not break the array reference
+// connection between the split transforms it creates. That's handled in
+// initialize, which creates a new empty array for each arrayAttr.
+function cloneTransforms(newTrace) {
```

Comment:
@alexcjohnson Never mind. I read my code more carefully. I'm just manually transferring the transforms. 👍